src/test/components/TrackContextMenu.test.js

```diff
@@ -534,8 +534,8 @@ describe('timeline/TrackContextMenu', function () {
 
       expect(getHumanReadableTracks(getState())).toEqual([
         'show [process]',
-        '  - show [thread DOM Worker] SELECTED',
         '  - hide [thread Style]',
+        '  - show [thread DOM Worker] SELECTED',
```

Comment:
I'm surprised by this change, I would have thought that `DOM Worker` would be above `Style`?

Reply:
aaaah actually this is because of the previous comment 😅 Obviously `thread DOM Worker` has index 0, and as a result it's always put at the end because of the faulty condition.